aiida/backends/tests/cmdline/commands/test_data.py

```diff
@@ -19,7 +16,6 @@
 import numpy as np
 import subprocess as sp
 
-from six.moves import cStringIO as StringIO
```

Comment:
Is there a ``from io import StringIO`` missing? Or maybe just put it as ``io.StringIO`` below?

Reply:
no, but there is `import io` at the top and just used `io.StringIO`

Reply:
Isn't there a bare ``StringIO`` [here](https://github.com/sphuber/aiida_core/blob/fix_3562_drop_python_2/aiida/backends/tests/cmdline/commands/test_data.py#L33)? I _think_ I'm looking at the right commit 😅

Reply:
yep you were right, but the function that it was in was never used :sweat_smile: the calls were commented out, so I will just remove the function